### ISSUE TITLE
[Toolbar] Bad direction of arrow if the search is active by default

### ISSUE THREAD

**Issue by xotahal** [xotahal/react-native-material-ui#97]
![untitled4](https://cloud.githubusercontent.com/assets/3531955/23250963/9be9674e-f9ab-11e6-9e6a-9198d6fadf31.gif)

**Comment by simsim0709** [comment]
When I look at `LeftElement` component's `constructor`, default `leftElement` state is `SEARCH_FORWARD_ICON` instead of `SEARCH_BACK_ICON`, if `isSearchActive` is true by default.
 
I think it's because of that..

```
constructor(props, context) {
        super(props, context);

        this.state = {
            styles: getStyles(this.props, this.context),
            leftElement: props.isSearchActive ? SEARCH_FORWARD_ICON : props.leftElement,
            spinValue: new Animated.Value(0),
        };
    }
```

**Comment by xotahal** [comment]
Fixed in 1.10
